openlibrary/plugins/openlibrary/js/ol.js

```diff
@@ -338,6 +338,24 @@ export default function init(){
             $('.wmd-preview').before('<h3 id="prevHead" style="margin:15px 0 10px;padding:0;">Preview</h3>');
         }
     });
+
+    // For Book Preview Button
+    $(document).ready(function(){
+        $('#bookPreviewButton').colorbox({
+            inline: true,
+            opacity: '0.5',
+            href: '#bookPreview'
+        })
+    })
+
+    $('.lazyIframe').show(function(){
+        // Find the iframes within our newly-visible element
+        $(this).find('iframe').prop('src', function(){
+            // Set their src attribute to the value of data-src
+            return $(this).data('src');
+        });
+    });
+
     initReadingListFeature();
     initBorrowAndReadLinks();
 }
```

Comment:
Please don't put these changes in ol.js - break them out into a separate file called previewButton and check for the existence of #bookPreviewButton in the page before executing.

index.js should look something like this:

```
if ( $( '#bookPreviewButton' ).length ) {
   require( './bookPreview' )();
}
```

Reply:
The discussion is done, but I think it might be best to "grandfather policy" this PR since the changes were happening simultaneously. Future PRs will have to follow that pattern :+1: